tasks/lammps/run.py

```diff
@@ -48,11 +49,35 @@ def _init_csv_file(csv_name):
 
 
 def _process_lammps_result(
-    lammps_output, result_file, num_procs, run_num, actual_time
+    lammps_output, result_file, num_procs, run_num, measured_time=None
 ):
-    print("Processing lammps output: \n{}\n".format(lammps_output))
-
-    reported_time = re.findall("Total wall time: ([0-9:]*)", lammps_output)
+    if "wasm" in result_file:
+        try:
+            result_json = json.loads(lammps_output, strict=False)
+        except json.decoder.JSONDecodeError:
+            print(
+                "WARNING: can't process result JSON for LAMMPS running w/ {} procs (run {})".format(
+                    num_procs, run_num
+                )
+            )
+            return
+        print(
+            "Processing lammps output: \n{}\n".format(
+                result_json["output_data"]
+            )
+        )
+        actual_time = (
+            float(int(result_json["finished"]) - int(result_json["timestamp"]))
+            / 1000
+        )
```

Comment:
This snippet is repeated. Can we have a utility function like `getTimeFromFaasmResultJson`?